### ISSUE TITLE
Cannot install with npm/yarn because dependency lib has incorrect github url

### ISSUE THREAD

**Issue by kaskelotti** [OverZealous/cdnizer#36]
Installing the package fails with error message

```
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads git://github.com/shahata/jsdelivr-cdn-data.git
Directory: /foo/bar/app
Output:
fatal: unable to connect to github.com:
github.com[0: 140.82.118.3]: errno=Operation timed out
```

There was an [issue](https://github.com/shahata/jsdelivr-cdn-data/issues/10) in the dependency package **jsdelivr-cdn-data** in which the package.json had incorrect format for the github url. This has been fixed with PR.

By looking at the package.json file of cdnizer, it seems to point directly to a specific commit of the dependency lib. To fix this issue, I would assume that changing the pointer to current HEAD of jsdelivr-cdn-data repo is enough.

**Comment by OverZealous** [comment]
@kaskelotti Thank you for reporting this. If you'd like to file a PR with the change, I'd be happy to look into merging it in and re-publishing the library.

I don't use this library in any of my own projects, so I'm not actively maintaining it right now.

**Comment by OverZealous** [comment]
Fix published as v3.2.0
